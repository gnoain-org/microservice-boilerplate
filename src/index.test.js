const request = require('supertest');
const app = require('./index');

describe('Server Index', () => {
  test('It should say hello', async () => {
    const response = await request(app.callback()).get('/');
    expect(response.text).toMatchSnapshot();
  });
});
