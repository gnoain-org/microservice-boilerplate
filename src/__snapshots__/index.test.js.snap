// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Index It should say hello 1`] = `"Hello World!!"`;
